README.md

```diff
@@ -59,35 +52,25 @@ If you would like to be notified of future conferences that would be added to th
-| [droidconIN](https://droidcon.in/2016/) | 10-11 Nov | MLR Convention Centre, Bengaluru | It is the biggest android conference in the region and is the place to be if you are into android development, design or architecture. |
-| [DevOps Days India](http://devopsdaysindia.org/) | 4-5 Nov | The Royal Orchind, Bengaluru | DevOpsDays India is a global event complementing other DevOpsDays events across the world. |
-| [UX India](http://www.2016.ux-india.org/) | 20-22 Oct | Westin, Hyderabad | India’s biggest international conference on user experience design. |
-| [PyCon India](https://in.pycon.org/2016/) | 24-25 Sept | Jawahar Lal Nehru University, New Delhi | The premier conference in India on using and developing the Python programming language. |
-| [Meta Refresh](https://metarefresh.in/2016/) | 17 Sept | MLR Convention Centre, Bengaluru | Meta Refresh is an event on front-end engineering and design. |
-| [JSFoo](https://jsfoo.in/2016/) | 15-16 Sept | MLR Convention Centre, Bengaluru | JSFoo is India’s premier JavaScript conference. |
-| [SANOG](http://www.sanog.org/sanog28/) | 1-9 Aug | Trident, BKC, Mumbai | The South Asian Network Operator Group. Engineers sharing operational knowledge on running ISPs and modern networks. |
-| [The Fifth Elephant](https://fifthelephant.in/2016/) | 28-29 July | NIMHANS Convention Centre, Bengaluru | The Fifth Elephant is India’s most renowned conference on data. |
-| [Rootconf](https://rootconf.in/2016/) | 14-15 Apr | MLR Convention Centre, Bengaluru | Rootconf is India’s principal conference where systems and operations engineers share real world knowledge. |
-| [IndiaHacks Conference](https://www.hackerearth.com/indiahacks-conference/) | 19 Mar | Vivanta by Taj, Bengaluru | IndiaHacks has an extensive lineup of speakers talking on a variety of technical subjects like ML, Big Data, AI etc. |
-| [RubyConf India](http://rubyconfindia.org/rci2016/) | 19-20 Mar | Le Meridien, Kochi |  RubyConf India is a global event complementing other RubyConf events across the world. |
-| [PyDelhi Conf](https://conference.pydelhi.org/2016/) | 5 Mar | Jawaharlal Nehru University, Delhi | PyDelhi conference is hosted annually by PyDelhi Community with an aim to promote Python programming language. |
-| [FOSSMeet](http://fossmeet.in/2016/) | 26-28 Feb | NIT Calicut | FOSSMeet is an annual event on Free and Open source software, conducted at National Institute of Technology, Calicut. |
-| [GopherCon India](http://www.gophercon.in/) | 19-20 Feb | Vivanta by Taj, Bengaluru | Go conference in India. Go is an open source project developed by a team at Google and many contributors from the open source community. |
-|[Mini Deb-Con](http://in2015.mini.debconf.org/)|30 - 31 Jan| Andheri (E), Mumbai | It is an awesome opportunity for any user who wants to experience the magic of FOSS |
+### 2018
```

Comment:
As a bullet point: `+ 2018 - [List of Conferences](year-pages/2018-conferences-list.md)` in one line.